test/repositories/slick/implementations/SlickAuthenticationRepositorySpec.scala

```diff
@@ -80,21 +88,36 @@ class SlickAuthenticationRepositorySpec extends AsyncWordSpec
   "SlickAuthenticationRepository#signUpUser" should {
 
     "signUp user" in {
-      val userAccess: UserAccess = genUserAccess.sample.value.copy(token = None)
-      Await.result(authenticationRep.signUpUser(userAccess), Duration.Inf)
-
-      val testQuery = db.run((for {
-        addressRow <- AddressesTable.all.filter(_.address === userAccess.address)
-        userRow <- UsersTable.all.filter(_.addressId === addressRow.addressId)
-        passwordRow <- PasswordsTable.all.filter(_.userId === userRow.userId)
-        tokenRow <- TokensTable.all.filter(_.tokenId === passwordRow.tokenId)
-      } yield (addressRow.address, passwordRow.password, userRow.firstName, userRow.lastName, tokenRow.token))
-        .result.headOption)
-
-      testQuery.map {
-        case Some(tuple) => assert(tuple._1 === userAccess.address &&
-          tuple._2 === userAccess.password && tuple._3 === userAccess.first_name.value &&
-          tuple._4 === userAccess.last_name.value && DataValidators.isValidUUID(tuple._5))
+      val userAccess: UserAccess = genUserAccess.sample.value
+      val key: String = config.get[String]("secretKey")
+      val algo = JwtAlgorithm.HS256
+
+      for {
+        createdtoken <- authenticationRep.signUpUser(userAccess)
```

Comment:
signUp or signIn?

Reply:
SignUp, that test is testing the signUp method. The repository no longer has a signIn method. The closest thing to it is the updateToken method.